Makefile

```diff
@@ -53,8 +54,11 @@ unittest:
 	@# "grep -n" returns the line number, then we print all successive lines with awk
 	@awk "NR>$$(grep -n 'Summary' luacov.report.out | cut -d':' -f1)" luacov.report.out
 
-$(DOCKER_COMPOSE): tox.ini
-	tox -e docker-compose --notest
+$(DOCKER_COMPOSE):
+	# From https://docs.docker.com/compose/install/#prerequisites
+	# docker-compose is a statically linked go binary, so we can simply download the binary and use it
```

Comment:
Interesting -- last I heard Docker Compose is actually written primarily in Python?

Downloading and inspecting the binary a little (e.g. using `strings`), seems like they've actually packaged the Python app and its dependencies into an executable, not rewritten it into Go.